ReactiveObjC/RACSignal+Operations.h

```diff
@@ -14,7 +14,7 @@
 @class RACMulticastConnection<__covariant ValueType>;
 @class RACScheduler;
 @class RACSequence<__covariant ValueType>;
-@class RACSubject;
+@class RACSubject<__covariant ValueType>;
```

Comment:
Thanks, I understand better now. I've now made `RACSubject` invariant instead of covariant in its generics. Do have another look when you have time!